prow/plugins/label/label.go

```diff
@@ -217,6 +227,35 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent, sc slackClient)
 		}
 	}
 
+	milestoneMaintainers, err := gc.ListTeamMembers(MILESTONE_MAINTAINERS_ID)
+	maintainersMap := map[string]bool{}
```

Comment:
May I suggest [sets.NewString](https://github.com/kubernetes/test-infra/blob/master/vendor/k8s.io/kubernetes/pkg/util/sets/string.go#L30)?

Reply:
If you're just checking membership then `map[string]bool` is idiomatic go. The sets package is useful if you need to do set intersections and the like, but otherwise we prefer to use maps.